content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -40,61 +118,109 @@ Count products
+
+If the :guilabel:`Counted Quantity` is greater than the :guilabel:`On Hand Quantity`, the value in
+the :guilabel:`Difference` column will be **green**. If the :guilabel:`Counted Quantity` is lesser
+than the :guilabel:`On Hand Quantity`, the value in the :guilabel:`Difference` column will be
+**red**. If the quantities match and haven't been changed at all, no value will appear in the
+:guilabel:`Difference` column.
+
+Subsequently, once applied, a move with the difference between the :guilabel:`On Hand Quantity` and
+the :guilabel:`Counted Quantity` will be recorded in the product's inventory adjustment history.
+
+.. image:: count_products/count-products-history-inventory-adjustments.png
    :align: center
    :alt: Inventory Adjustments History dashboard detailing a list of prior product moves.
 
-.. note::
+.. important::
    Sometimes a count occurs, but can not be applied in the database right away. In the time between
    the actual count and applying the inventory adjustment, product moves can occur. In that case,
-   the :guilabel:`On Hand Quantity` in the database can change and will not be consistent with the
-   counted quantity. As an extra caution measure, Odoo will ask for confirmation before applying
-   the inventory adjustment.
+   the on hand quantity in the database can change and will not be consistent with the counted
+   quantity. As an extra caution measure, Odoo will ask for confirmation before applying the
+   inventory adjustment.
 
-Plan counts
-===========
+Change inventory count frequency
+================================
 
-Each inventory adjustment line contains the following information:
+By default, the *scheduled date* for inventory adjustments are always scheduled for the 31st of
+December of the current year (:dfn:`12/31/2023` in the database). However, for some companies, it
+is crucial that they have an accurate inventory count at all times. In such cases, the default
+scheduled date can be modified.
 
-- :guilabel:`Scheduled Date`: the date at which a count should be made.
-- :guilabel:`User`: the person in charge of the count.
-- :guilabel:`Accounting Date`: the date at which the adjustments will be accounted. The column is
-  hidden by default, but can be made visible by opening the column options icon.
+To modify the default scheduled date, go to :menuselection:`Inventory app --> Configuration -->
+Settings`, and scroll to the :guilabel:`Operations` section. Locate the :guilabel:`Annual Inventory
+Day and Month` setting field, which includes a drop-down that is set to `31 December` by default.
 
-.. important::
-   In the Barcode app, users can only view counts assigned to them that are scheduled for today or
-   earlier.
+.. image:: count_products/count-products-annual-inventory.png
+   :align: center
+   :alt: Adjust the next inventory count date with the Annual Inventory Day and Month setting.
+
+To change the day, click :guilabel:`31`, and change it to a day within range `1-31`, depending on
```

Comment:
might want to specify that you type in the numbers, it's not a drop-down like the month?